src/CMakeLists.txt

```diff
@@ -93,6 +93,25 @@ target_link_libraries(bitcoin_common
 )
 
 
+if(ENABLE_WALLET)
+  add_subdirectory(wallet)
+
+  if(BUILD_WALLET_TOOL)
+    add_executable(bitcoin-wallet
+      bitcoin-wallet.cpp
+      init/bitcoin-wallet.cpp
+      wallet/wallettool.cpp
+    )
+    target_link_libraries(bitcoin-wallet
+      bitcoin_wallet
```

Comment:
The naming and semantics between `-` and `_` constantly trips me up, but I don't think there is really a viable alternative.